.eslintrc.js

```diff
@@ -170,6 +170,21 @@ module.exports = {
           "Do not import default from React. Use a namespace import (import * as React from 'react';) instead.",
         selector: 'ImportDeclaration[source.value="react"] ImportDefaultSpecifier',
       },
+      {
+        message:
+          "Do not import default from ReactDOM. Use a namespace import (import * as ReactDOM from 'react-dom';) instead.",
+        selector: 'ImportDeclaration[source.value="react-dom"] ImportDefaultSpecifier',
```

Comment:
`selector:
          'ImportDeclaration[source.value=/^react-dom(\\u002F(server|client))?$/] ImportDefaultSpecifier'` – this could work with only one additional entry using a regex but its clearer to separate out all the error messages